### ISSUE TITLE
Select: using autocomplete but setting aria-autocomplete to none

### ISSUE THREAD

**Issue by Sidnioulz** [radix-ui/primitives#3631]
### Current Behavior

Select implements an inline autocomplete algorithm. Yet, it sets `aria-autocomplete` to `none`.

### Expected behavior

The `aria-autocomplete` attribute should be set to `inline` instead.

### Reproducible example

https://www.radix-ui.com/primitives/docs/components/select

### Suggested solution

Set `aria-autocomplete` to `inline`

### Additional context

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Attributes/aria-autocomplete

### Your environment

| Software         | Name(s) | Version |
| ---------------- | ------- | ------- |
| Radix Package(s) | 1.4.2   |         |
| React            | n/a     |         |
| Browser          | n/a     |         |
| Assistive tech   | n/a     |         |
| Node             | n/a     |         |
| npm/yarn/pnpm    | n/a     |         |
| Operating System | n/a     |         |
